README.md

```diff
@@ -16,7 +16,7 @@ searchCtrl.addTo(map);
 Then load your GeoJSON layer and index the features, choosing the properties you want to index
 (note you can pass either the FeatureCollection itself or its .features), e.g.
 <pre>
-searchCtrl.indexFeatures(jsonData, ['name', 'company', 'details'];
+searchCtrl.indexFeatures(jsonData, ['name', 'company', 'details']);
 </pre>
 
 Finally you need to bind each layer (marker) to the feature it is associated with, so that selecting an item in the result list opens up the matching popup :
```

Comment:
I'm new to leaflet and can't make head nor tails of the line `feature.layer = layer`. Hence my result-items don't get the class `clickable`. Could you please help me? My [code](https://github.com/lacemap/lacemap/blob/gh-pages/map-config.js)